index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+      <div class="nav">
+        <header>
+            <a href="#" class="logo"><h1>Welcome</h1></a>
+        </header>
+        <nav>  
+            <a href="#"><img  src="images/Icons/Menu.png" alt="togglebutton" class="MenuIcons"></a>
+        </nav>
+      </div> 
+      <main>
+        <section class="section-1">
+             
+                <h2 id="Primary-1">Hey there. I’m Mariane</h2>
```

Comment:
- [ ] Change `<h2>` to `<h1>` 🍀. Use h1 for important things. With doing that you will improve SEO, for example, if someone search your website they also will be able to see your name 🔍.